crates/holochain_zome_types/src/zome.rs

```diff
@@ -168,10 +188,12 @@ pub struct CoordinatorZomeDef(ZomeDef);
 /// The serialized form of a ZomeDef, which is identical for Wasm zomes, but
 /// unwraps InlineZomes to just a bare network seed.
 #[derive(Serialize)]
-#[serde(untagged)]
+#[serde(tag = "type")]
```

Comment:
this change is a breaking change to the DB schema I think... maybe it can be handled via migrations?

Reply:
@Connoropolous is there any way around this? Can you explain why it's necessary?

Reply:
Ah I see, you're adding a new variant altogether. I commented too soon.